src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/LoansToDirectorsAdditionalInfoControllerTest.java

```diff
@@ -82,6 +88,8 @@
     private static final String MOCK_CONTROLLER_PATH = UrlBasedViewResolver.REDIRECT_URL_PREFIX + "mockControllerPath";
 
 
+    private static final String COMPANY_ACCOUNTS_DATA_STATE = "companyAccountsDataState";
```

Comment:
Please move this constant (and the others while you're at it) to the top, above the other vars.